samples/examples/image_classifier/infer.md

```diff
@@ -121,479 +117,620 @@ def show(X):
-        - name: cifar10-outlier
-        - name: cifar10-drift
-          batch:
-            size: 20
-      output:
-        steps:
-        - cifar10
-        - cifar10-outlier.outputs.is_outlier
-
+```bash
+cat ../../pipelines/cifar10.yaml
+```
 
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: cifar10-production
+spec:
+  steps:
+    - name: cifar10
+    - name: cifar10-outlier
+    - name: cifar10-drift
+      batch:
+        size: 20
+  output:
+    steps:
+    - cifar10
+    - cifar10-outlier.outputs.is_outlier
 
-```python
-!seldon pipeline load -f ../../pipelines/cifar10.yaml
 ```
 
-    {}
+```bash
+seldon pipeline load -f ../../pipelines/cifar10.yaml
+```
 
+```json
+{}
 
+```
 
-```python
-!seldon pipeline status cifar10-production -w PipelineReady| jq -M .
-```
-
-    {
-      "pipelineName": "cifar10-production",
-      "versions": [
-        {
-          "pipeline": {
-            "name": "cifar10-production",
-            "uid": "ccvfa5t4nntrbfkuk5hg",
-            "version": 2,
-            "steps": [
-              {
-                "name": "cifar10"
-              },
-              {
-                "name": "cifar10-drift",
-                "batch": {
-                  "size": 20
-                }
-              },
-              {
-                "name": "cifar10-outlier"
-              }
-            ],
-            "output": {
-              "steps": [
-                "cifar10.outputs",
-                "cifar10-outlier.outputs.is_outlier"
-              ]
-            },
-            "kubernetesMeta": {}
-          },
-          "state": {
-            "pipelineVersion": 2,
-            "status": "PipelineReady",
-            "reason": "created pipeline",
-            "lastChangeTimestamp": "2022-10-06T15:32:39.926431438Z"
-          }
-        }
-      ]
-    }
+```bash
+seldon pipeline status cifar10-production -w PipelineReady| jq -M .
+```
 
+```json
+{
+  "pipelineName": "cifar10-production",
+  "versions": [
+    {
+      "pipeline": {
+        "name": "cifar10-production",
+        "uid": "ccvfa5t4nntrbfkuk5hg",
+        "version": 2,
+        "steps": [
+          {
+            "name": "cifar10"
+          },
+          {
+            "name": "cifar10-drift",
+            "batch": {
+              "size": 20
+            }
+          },
+          {
+            "name": "cifar10-outlier"
+          }
+        ],
+        "output": {
+          "steps": [
+            "cifar10.outputs",
+            "cifar10-outlier.outputs.is_outlier"
+          ]
+        },
+        "kubernetesMeta": {}
+      },
+      "state": {
+        "pipelineVersion": 2,
+        "status": "PipelineReady",
+        "reason": "created pipeline",
+        "lastChangeTimestamp": "2022-10-06T15:32:39.926431438Z"
+      }
+    }
+  ]
+}
 
+```
 
 ```python
 infer("cifar10-production.pipeline",20, "normal")
 ```
 
+<!-- FIXME All these extra ``` are erroneous and added by my scripts -->
```

Comment:
:wrench: As this comment says, these extra block delimiters are erroneous.  I'm guessing a terminating delimiter condition isn't correctly clearing the hold state.

Reply:
These are added by the `format-output-blocks.sed` script specifically.
It turns out they _are_ outputs from the `infer` calls and, although it isn't easy to see in GitHub diffs, the links are wrapped with lines of four leading spaces, which is detected by the script as an output block.
The links themselves are not indented, thus terminate the output block, but the line following each again looks like an output block and ends up creating empty outputs.

So far as I can see, the beginning and terminating delimiters _are_ correctly matched, so this is more noise rather than being genuinely incorrect output.

I can try adding an exception for bangs/plings at the start of a line when in an output block to reduce the noise.